types/office-js/index.d.ts

```diff
@@ -17176,9 +17176,12 @@ declare namespace OfficeExtension {
 ////////////////////////////////////////////////////////////////
 
 
+
```

Comment:
FYI spacing added here so spacing is consistent with the rest of the file, I believe. Thanks